### PR TITLE
Avoid decimals call when calling handleFees on FeeBurner

### PR DESCRIPTION
**Pull request by manhlx3006**

### CODE REVIEW

**Comment by ilanDoron** [comment]
fixes #437

**Comment by manhlx3006** [comment]
Yes, checked from `feeBurner` test (only call `handleFees` func), gas reduces around 3000+

File: contracts/FeeBurner.sol [hunk 1]
```diff
@@ -109,7 +109,7 @@ contract FeeBurner is Withdrawable, FeeBurnerInterface, Utils2 {
         require(msg.sender == address(kyberNetwork));
         require(tradeWeiAmount <= MAX_QTY);
 
-        uint kncAmount = calcDestAmount(ETH_TOKEN_ADDRESS, ERC20(knc), tradeWeiAmount, kncPerEthRatePrecision);
+        uint kncAmount = calcDestAmountWithDecimals(ETH_DECIMALS, MAX_DECIMALS, tradeWeiAmount, kncPerEthRatePrecision);
```
Comment (tranvictor) [inline contracts/FeeBurner.sol]: lets have a comment explaining that `MAX_DECIMAL` equals to decimal of KNC

Comment (manhlx3006) [reply contracts/FeeBurner.sol]: PR to add the comment here: https://github.com/KyberNetwork/smart-contracts/pull/489

**Pull request merged**